com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/LitSurfaceInputsUIBlock.cs

```diff
@@ -10,21 +10,32 @@
 
 namespace UnityEditor.Rendering.HighDefinition
 {
-    // This block is shared for Lit and Layered surface inputs + tesselation variants
-    class LitSurfaceInputsUIBlock : MaterialUIBlock
+    /// <summary>
+    /// Lit surface inputs material UI block.
+    /// This block is shared for Lit and Layered surface inputs + tesselation variants.
+    /// </summary>
+    public class LitSurfaceInputsUIBlock : MaterialUIBlock
     {
+        /// <summary>Lit Surface Inputs Features.</summary>
         public enum Features
         {
+            /// <summary>Minimal Lit Surface Inputs fields.</summary>
             None            = 0,
+            /// <summary>Coat Mask fields.</summary>
```

Comment:
Displays Coat Mask fields.

Can you add **Displays** to the two below and change **Show** to **Displays** for the rest?